lxd/instance/drivers/driver_qemu.go

```diff
@@ -4392,8 +4403,7 @@ func (d *qemu) IsPrivileged() bool {
 	return false
 }
 
-// Snapshot takes a new snapshot.
-func (d *qemu) Snapshot(name string, expiry time.Time, stateful bool) error {
+func (d *qemu) takeSnapshot(name string, expiry time.Time, stateful bool) error {
```

Comment:
@tomp this will collision with the `snapshot` boolean field in `common`

Reply:
Ah OK, can we change the snapshot bool to `isSnapshot` it would actually read more clearly in my view (as "snapshot" can be an action as well as a description).